crates/yas_core/src/tests/test_contracts/test_yas_pool.cairo

```diff
@@ -486,81 +491,1014 @@ mod YASPoolTests {
+                    //expect(liquidityGross).to.eq(0) -done
+                    //expect(feeGrowthOutside0X128).to.eq(0) -done
+                    //expect(feeGrowthOutside1X128).to.eq(0) -done
+                    //;({ liquidityGross, feeGrowthOutside0X128, feeGrowthOutside1X128 } = await pool.ticks(-tickSpacing)) -done
+                    //expect(liquidityGross).to.eq(250) -done
+                    //expect(feeGrowthOutside0X128).to.eq(0) -done
+                    //expect(feeGrowthOutside1X128).to.eq(0) -done
+
+                    let (yas_pool, token_0, token_1, mint_callback, min_tick, max_tick) = setup();
+                    mint_callback
+                        .mint(
+                            yas_pool.contract_address,
+                            WALLET(),
+                            IntegerTrait::<i32>::new(240, true),
+                            IntegerTrait::<i32>::new(0, false),
+                            100
+                        );
+                    mint_callback
+                        .mint(
+                            yas_pool.contract_address,
+                            WALLET(),
+                            IntegerTrait::<i32>::new(tick_spacing(FeeAmount::MEDIUM), true),
+                            IntegerTrait::<i32>::new(0, false),
+                            250
+                        );
+                    //burn
+                    assert(1 == 2, 'burn() func doesnt exist yet');
+
+                    let tick_info = yas_pool.ticks(IntegerTrait::<i32>::new(240, true));
+                    assert(tick_info.liquidity_gross == 0, 'wrong liquidity_gross amount');
+                    assert(
+                        tick_info.fee_growth_outside_0X128 == 0, 'wrong fee_growth_outside_0X128'
+                    );
+                    assert(
+                        tick_info.fee_growth_outside_1X128 == 0, 'wrong fee_growth_outside_1X128'
+                    );
+
+                    let tick_info = yas_pool.ticks(-yas_pool.tick_spacing());
+                    assert(tick_info.liquidity_gross == 250, 'wrong liquidity_gross amount');
+                    assert(
+                        tick_info.fee_growth_outside_0X128 == 0, 'wrong fee_growth_outside_0X128'
+                    );
+                    assert(
+                        tick_info.fee_growth_outside_1X128 == 0, 'wrong fee_growth_outside_1X128'
+                    );
                 }
-            // TODO: 'max tick with max leverage'
-            // TODO: 'works for max tick'
-            // TODO: 'removing works'
-            // TODO: 'adds liquidity to liquidityGross'
-            // TODO: 'removes liquidity from liquidityGross'
-            // TODO: 'clears tick lower if last position is removed'
-            // TODO: 'clears tick upper if last position is removed'
-            // TODO: 'only clears the tick that is not used at all'
```

Comment:
Having the cases that are yet to be implemented can be useful. Let's make sure that if we delete them, it's because they are implemented

Reply:
I agree. Those comments were removed because they were transformed into empty test functions with the appropriate info, ensuring a clear record of pending test cases while also simplifying their development